src/Dfe.PlanTech.AzureFunctions/host.json

```diff
@@ -10,7 +10,6 @@
     }
   },
   "extensions": {
-    "serviceBus": {},

```

Comment:
Whoops, forgot `host.json` is included in the source control. This shouldn't have been checked in; not that it would have made a difference as you've seen!

Reverted.